ict-water-flow/transformers/main/source.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Transforms an ICT P-Node counter representing a count of the amount of 10L units of water
+ * to pass through a flow meter within a 15 minute period.
+ *
+ * The record is laid out as follows:
+ * 
+ *  0  - 3   system time
+ *  4  - 5   battery level
+ *  6  - 9   big endian count 1
+ * 10 - 13   big endian count 2
+ * 14 - 17   big endian count 3
+ * 18 - 21   big endian count 4
+ */
+function transform(time, nwkAddr, fPort, payload) {
+  var payloadBytes = atob(payload);
```

Comment:
Could you use const instead of var for value declarations? (Assuming the JS runtime this is targeting supports that)

Reply:
(or "let" for mutable values)

Reply:
Thanks. I should be able to get away with `const`.